docs/build_mozc_in_windows.md

```diff
@@ -40,6 +44,8 @@ Building Mozc on Windows requires the following software.
     * `six`
     * `requests`
   * `.NET 6` or later (for `dotnet` command).
+  * [Bazel](https://bazel.build/) (If you are building Mozc with Bazel. However, you do not need to install a new JDK just for Mozc)
```

Comment:
I'd like to keep this section for GYP. Let's make a section under `## Build with Bazel (experimental)`.

Notes for JDK and MSYS2 are helpful, but they are not parts of build steps. How about making a section about `Tips for Bazel setup` to the end of `## Build with Bazel (experimental)`?